apps/astraplusplus/widget/DAO/Proposal/Create/FunctionCall.jsx

```diff
@@ -107,9 +107,16 @@ const handleFunctionCall = () => {
       });
       return;
     }
+    if (state.gas > 300) {
```

Comment:
as mentioned 300 TGas will always fail. It's difficult to give an exact estimate, but this is the gas that gets attached to the cross contract call, so it's always less than 300 TGas. I would reduce the max allowed Gas here to the default 270

Reply:
Quick question
Is it necessary it will always be a cross contract call.?

Reply:
Yes that's by design of Sputnik DAO contract

Reply:
I will update it, thanks for the information